bin/all_sky_search/pycbc_coinc_statmap_inj

```diff
@@ -164,6 +171,9 @@ if len(zdata) > 0:
         ifar[i] = background_time / (fnlouder[i] + 1)
         fap[i] = 1 - numpy.exp(- coinc_time / ifar[i])
         logging.info('processed %s, %s' % (i, fstat))
+    else:
```

Comment:
The code will *always* end up in this else block though! (Perhaps avoid `for/ else` unless it's needed as the syntax here is confusing). I also thought that the idea was *not* to populate the `foreground/ifar` and `foreground/fap` columns if skipping exclusive, which is not what is being done here?

Reply:
I think it won't unless the flag is set

I've put another if statement in to make it more obvious than a for/else